modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/extensionflow/GeoExtensionTestCase.java

```diff
@@ -0,0 +1,142 @@
+package org.wso2.carbon.integration.test.extensionflow;
```

Comment:
@keizer619 
Thank you for pointing out. Fixed in PR: https://github.com/wso2/product-cep/pull/186